conda-recipes/pypng-feedstock/ci_support/local_build.sh

```diff
@@ -0,0 +1,50 @@
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
```

Comment:
Is this file needed?

Reply:
No, I mistakenly added that. Removing it.